README.rst

```diff
@@ -3,18 +3,18 @@ django-push-notifications
 .. image:: https://api.travis-ci.org/jleclanche/django-push-notifications.png
 	:target: https://travis-ci.org/jleclanche/django-push-notifications
 
-A minimal Django app that implements Device models that can send messages through APNS, GCM and WNS.
+A minimal Django app that implements Device models that can send messages through APNS, FCM/GCM and WNS.
 
 The app implements three models: ``GCMDevice``, ``APNSDevice`` and ``WNSDevice``. Those models share the same attributes:
  - ``name`` (optional): A name for the device.
  - ``active`` (default True): A boolean that determines whether the device will be sent notifications.
  - ``user`` (optional): A foreign key to auth.User, if you wish to link the device to a specific user.
  - ``device_id`` (optional): A UUID for the device obtained from Android/iOS/Windows APIs, if you wish to uniquely identify it.
- - ``registration_id`` (required): The GCM registration id or the APNS token for the device.
+ - ``registration_id`` (required): The FCM registration id or the APNS token for the device.
```

Comment:
There is so formatting incongruity here.  On line 6 you use FCM/GCM, while on this and subsequent lines you use only FCM.  Should probably keep it FCM/GCM.